settings.php

```diff
@@ -115,6 +115,11 @@
             get_string('lowlicenses', 'mod_zoom'), 0, 1);
     $settings->add($utmost);
 
+    $instanceusers = new admin_setting_configcheckbox('zoom/instanceusers',
+                    get_string('instanceusers', 'mod_zoom'),
+                    get_string('instanceusers_desc', 'mod_zoom'), 0);
```

Comment:
These may be the default values, but I think it may be important for us to be clear that we want this off by default. Also, fixes the extra indentation.

```suggestion
            get_string('instanceusers', 'mod_zoom'),
            get_string('instanceusers_desc', 'mod_zoom'), 0, 1, 0);
```